src/components/sidebarContent/bin/SidebarBin.vue

```diff
@@ -90,10 +103,18 @@ export default {
 <template>
   <div class="sidebar-bin">
     <div v-if="deletedLists.length || deletedItems.length">
-      <SectionCard
-        v-show="deletedLists.length"
-        title="lists"
-      >
+      <div class="bin-mode-buttons">
+        <ButtonText
+          v-for="(type, index) in ['lists', 'items']"
+          v-show="[type === 'lists' ? 'deletedLists' : 'deletedItems'].length"
```

Comment:
it's better to make computed property with already filtered list and pass it here in v-for

also you don't need index, you can use type as a key
also in general using index as a key it's just workaround for linter, it doesn't make sense